client/lib/sections-utils/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External Dependencies
+ */
+import escapeRegExp from 'lodash/escapeRegExp';
```

Comment:
nowadays we always try to use the named import syntax like:
`import { escapeRegExp } from 'lodash'`